app/tag/[category]/page.tsx

```diff
@@ -12,46 +10,53 @@ export async function generateMetadata({
   params: { category: string };
 }): Promise<Metadata> {
   var title = params.category.replaceAll("-", " ");
-  title = title[0].toUpperCase() + title.slice(1, title.length)
-  if (!title) {
-    return {
-      title: "Not found",
-      description: `${title} not found`,
-    };
-  }
-  return {
-    title: title,
-    description: title,
-  };
+  title = title[0].toUpperCase() + title.slice(1, title.length);
+  if (title) return { title: title, description: title };
+
+  return { title: "Not found", description: `${title} not found` };
 }
 
 async function page({ params }: { params: { category: string } }) {
-  const category = params.category;
-  var heading = params.category.replace("-", " ");
-  heading = heading[0].toUpperCase() + heading.slice(1, heading.length)
+  const category = params.category.replaceAll(" ", "-");
+
+  var heading = category.replace("-", " ");
+  heading = heading[0].toUpperCase() + heading.slice(1, heading.length);
 
-  const [recommened, latest]: [Article[], Article[]] = await Promise.all([
-    fetchRecommendedArticles(category),
-    fetchNewestArticles(category, 0),
-  ]);
-  
-  
+  const recommended: { recommended: Article[] } =
+    await fetchRecommendedArticles(category);
+  const newest: { newest: Article[] } = await fetchNewestArticles(category);
 
   return (
     <div className="mx-2">
-      <h1 className="text-4xl md:text-5xl text-center font-semibold">
+      <h1 className="text-4xl md:text-5xl text-center font-semibold capitalize">
         {heading}
       </h1>
-      <Suspense fallback={[1, 2].map(() => Skeleton({ image: true }))}>
-        <Recommended recommended={recommened} />
-      </Suspense>
-      <br />
-      <hr />
-      <br />
-      <Suspense fallback={latest.map(() => Skeleton({ image: true }))}>
-        <h3 className="mb-3 text-2xl font-semibold">Latest</h3>
-        <Latest latest={latest} />
-      </Suspense>
+      {recommended.recommended.length > 0 ? (
+        <>
+          <br />
+          <hr />
+          <br />
+          <h1 className="mb-3 text-2xl font-semibold">Recommended stories</h1>
+          <Recommended recommended={recommended} />
+        </>
+      ) : (
+        <h1 className="mb-3 text-2xl font-semibold">
+          No recommended stories found!
+        </h1>
+      )}
+      {newest.newest.length > 0 ? (
+        <>
+          <br />
```

Comment:
Why do we need <br/>, can we do the same thing with css?

Reply:
Sure, i will remove all `<br /> and <hr />`s later on. All `<br />` were just for a quick test on building time